src/components/Charts/ZoneLinePath.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import RectClipGroup from './RectClipGroup';
+import { curveNatural } from '@vx/curve';
+import { CurveFactory } from 'd3-shape';
+import { LinePath } from '@vx/shape';
+import { Region } from './utils';
+
+const ZoneLinePath = <T extends unknown>({
```

Comment:
hmmm

Reply:
no good? I want to be able to pass different types of points to it, I thought this was the best practice 😂

Reply:
This is used for `ChartRt` whose points have a different shape than the points for the Positive test rate chart